docs/source/releasenotes.rst

```diff
@@ -1,9 +1,66 @@
 Release notes
 =============
 
+.. note::
```

Comment:
I placed this on top as a sticky note since I have the feeling we want this "pinned" on top of everything and it's not bound to a release per se.